src/constants.js

```diff
@@ -1,4 +1,15 @@
 import { camelCase } from 'change-case'
+import { version } from '../package.json'
+
+export const SAUCE_CONNECT_VERSION = '4.5.4'
```

Comment:
Version 4.6.x is coming soon, isn't there a way to download the latest automatically instead of hardcoding it here?

Reply:
we could do that if there is an endpoint that exposes the latest version number, are you aware of such an endpoint?

Reply:
honestly though I feel like we should make a release for every sauce connect release so that people have the choice to pick their version based on the node package version.